frontend/css/playlists.less

```diff
@@ -277,3 +278,25 @@
     box-shadow: 0 0 0 2px rgba(53, 48, 112, 0.5);
   }
 }
+
+.wikipedia-extract {
```

Comment:
I think we should rename this class, it made sense before but now that we use it in the playlist page it should really be renamed.
Obviously not something you added yourself, but since we're in here modifying this specific piece of css let's improve the naming.

I suggest `text-extract` or `text-summary` instead, but open to suggestions if you have a better idea.
You'll need to do a search+replace all and change every instance.